### ISSUE TITLE
FTBU Max Claims Compatability

### ISSUE THREAD

**Issue by ghost** [ForgeEssentials/ForgeEssentials#2387]
Hey so i put an issue on FTBU for you as asked and adding it here for you 

https://github.com/FTBTeam/FTB-Utilities/issues/916

**Comment by Zethalion** [comment]
Please follow our guidelines for reporting issues https://github.com/ForgeEssentials/ForgeEssentials#reporting-bugs

Edit your post

**Comment by spacebuilder2020** [comment]
A ftbu compatibility layer has been added that addresses https://github.com/FTBTeam/FTB-Utilities/issues/916#issuecomment-453496927

Closing this issue!

**Comment by spacebuilder2020** [comment]
@MacWhinny Please Post your latest.log file

**Comment by ghost** [comment]
https://hastebin.com/ubucilezuq.md
https://hastebin.com/yivawozaru.sql
https://hastebin.com/acehocejaz.md
https://hastebin.com/xepuwuhati.md
https://hastebin.com/etetawasow.md

could it have something to do with preassigned claim amounts. the default rank _GUEST_ max claims went to 100 but i tried to set them to 1 before a restart

**Comment by spacebuilder2020** [comment]
Not sure.  I was able to set permissions for _OPS_ and have the claims show up.

Please post your permissions.json file

**Comment by ghost** [comment]
https://pastebin.com/EpMYb6h5

I don't know if it should look like that, but i've added perms but do not see any i have added.

**Comment by spacebuilder2020** [comment]
It is rather clear that for some reason, the permissions are not sticking.  Try adding them manually to the permissions section for the ops group.

**Comment by ghost** [comment]
Is this the way it;s supposed to be in the configs "ftbutilities.claims.max_chunks": "10"

**Comment by ghost** [comment]
https://hastebin.com/oyevoyazuj.cs

this is what i get when a player claims a chunk, they have the default 100 claims and 50 chunkloading claims even after i set the max claims to 10, i asked them to remove all of their claims before claiming a chunk. 

FTBU ranks are set to false.

**Comment by spacebuilder2020** [comment]
Please post your updated permission.json file again

**Comment by ghost** [comment]
https://hastebin.com/zufuvibuba.json

**Comment by spacebuilder2020** [comment]
Please update forge to 2784, ftblib to 5.4.1.75, and ftbu to 5.3.0.72 and retest.

Do this both on the server and the client.

All previous versions are currently not supported and not guaranteed to work.

I can work on supporting older versions in a future commit.

**Comment by spacebuilder2020** [comment]
After additional testing with ftbu version 5.3.0.52 and ftb lib 5.3.0.56 in an isolated environment, I have determined that the max claims permission is being registered correctly.  I also tested with AM3 v1.0.1 (modpack that @MacWhinny was using) and determined that they don't get registered there.  I am closing this issue because AM3 compatibility is beyond the scope of this issue.

@MacWhinny Please create a new issue to address AM3 compatibility following rules for reporting bugs here. https://github.com/ForgeEssentials/ForgeEssentials#reporting-bugs

**Comment by 1time** [comment]
I thought that we should open this Issue since I faced exactly the same issue on a different mod pack. 
After thorough testing the issue is caused when EnderIO is present on the modpack.

The way I tested it was that I compared the 2 packs and I found the common mods then I ve set up a server with those mods +FTBU and FE and I started taking mods off the server, as soon as I took enderIO off the ftbutilities.chunkloader.max_chunks and the ftbutilities.claims.max_chunks were updating via FE. as they did when I had only forge ftbu and FE installed.

Any help on that matter would be much appreciated. Since this "incompatibility" is preventing the use of both mods at the same time.

All tests were done with forge 2808, FE release 44, and FTBU 5.3.079 & EnderIO 5.0.40
